woof-code/rootfs-skeleton/usr/share/doc/HOWTO_dialup.htm

```diff
@@ -1,11 +1,13 @@
 <!DOCTYPE html>
 <html>
 <head>
-	<title>HOWTO Internet</title>
+	<title>HOWTO Internet by Dialup</title>
```

Comment:
Is this page still relevant and useful?

Reply:
IDK. It was already there so I put it in, though it hasn't been updated.

Reply:
@peabee, @01micko do you have any idea what to do with this file?

Reply:
I think this file is somewhat useless according to current internet standards. I don't think anyone uses dial-up these days. So, I am deciding to delete this file.